compiler/rustc_llvm/llvm-wrapper/PassWrapper.cpp

```diff
@@ -875,8 +875,12 @@ LLVMRustOptimizeWithNewPassManager(
 #if LLVM_VERSION_GE(11, 0)
       OptimizerLastEPCallbacks.push_back(
         [Options](ModulePassManager &MPM, OptimizationLevel Level) {
+#if LLVM_VERSION_GE(14, 0)
+          MPM.addPass(ModuleMemorySanitizerPass(Options));
+#else
           MPM.addPass(MemorySanitizerPass(Options));
           MPM.addPass(createModuleToFunctionPassAdaptor(MemorySanitizerPass(Options)));
+#endif
```

Comment:
This endif should be one line higher, the function pass needs to run on LLVM 14 as well.

Reply:
Done, thanks. I thought I had tried that, but evidently not.